src/3box.js

```diff
@@ -23,6 +23,8 @@ const utils = require('./utils/index')
 const idUtils = require('./utils/id')
 const config = require('./config.js')
 const API = require('./api')
+const IPFSRepo = require('ipfs-repo')
+const LevelStore = require('datastore-level/src/index.js')
```

Comment:
Why are we requiring the file directly?

Reply:
yeah can fix that, think left over from when I reference browser implementation specifically